bin/stacks/routing-dashboard-stack.ts

```diff
@@ -609,6 +611,26 @@ export class RoutingDashboardStack extends cdk.NestedStack {
                 title: 'Total Requests/Responses',
               },
             },
+            {
+              height: 6,
+              width: 24,
+              type: 'metric',
+              properties: {
+                metrics: REQUEST_SOURCES.map((source) => [
+                  'Uniswap',
+                  `GET_QUOTE_REQUEST_SOURCE: ${source}`,
+                  'Service',
+                  'RoutingAPI',
+                  { label: `Source: ${source}` },
```

Comment:
nit, probably we dont need the `Source:`

Reply:
Fixed.